client/components/account-details/banner.tsx

```diff
@@ -0,0 +1,65 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { ExternalLink } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import InlineNotice from 'wcpay/components/inline-notice';
+import {
+	AccountDetailsData,
+	BannerBackgroundColor,
+} from 'wcpay/types/account/account-details';
+import { getIconByName } from './utils';
+
+interface BannerProps {
+	banner: AccountDetailsData[ 'banner' ];
+}
+
+const getNoticeStatusFromColor = ( color: BannerBackgroundColor ) => {
+	switch ( color ) {
+		case 'green':
+			return 'success';
+		case 'blue':
+			return 'info';
+		case 'yellow':
+			return 'warning';
+		case 'red':
+			return 'error';
+		default:
+			return 'info';
+	}
+};
+
+const Banner: React.FC< BannerProps > = ( { banner } ) => {
+	if ( ! banner ) {
+		return null;
+	}
+
+	return (
+		<InlineNotice
+			status={ getNoticeStatusFromColor( banner.background_color ) }
+			icon={ getIconByName( banner.icon ) }
+			className="woopayments-account-details__banner"
+			isDismissible={ false }
+		>
+			<div>
+				{ banner.text }
+				{ banner.cta_text && banner.cta_link && (
+					<>
+						{ ' ' }
+						<ExternalLink href={ banner.cta_link }>
+							{ banner.cta_text }
+						</ExternalLink>
+					</>
+				) }
+			</div>
```

Comment:
Is the property-less `div` needed here?


```suggestion
			<>
				{ banner.text }
				{ banner.cta_text && banner.cta_link && (
					<>
						{ ' ' }
						<ExternalLink href={ banner.cta_link }>
							{ banner.cta_text }
						</ExternalLink>
					</>
				) }
			</>
```

Reply:
Either that, or maybe you can switch to a paragraph? Not sure if it would help with anything at all, but I always try to have some text-related element around... text.

Reply:
Oh, you're totally right here. Applied your suggestion 34795e89ac44e8aab56463f8893911abe2a6e9c4